tools/webpack/blocks.js

```diff
@@ -238,12 +245,14 @@ module.exports = [
 					vendors: {
 						name: 'vendors',
 						test: /[\\/]node_modules[\\/]/,
```

Comment:
I'd like to change this to the list of dependencies, even if we have to do it manually. Maybe in another PR :)

Reply:
If I recall correctly, @DAreRodz tried listing dependencies, but you also need to list their dependencies and so on. It might be difficult to maintain, but it's worth trying.